### PR TITLE
HPCC-14247 Document FOLD / NOFOLD options for SERVICE

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@ghalliday Please review

### CODE REVIEW

File: docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml [hunk 1]
```diff
@@ -162,6 +168,28 @@
             </row>
 
             <row>
+              <entry><emphasis>FOLD<indexterm>
+                  <primary>FOLD</primary>
```
Comment (richardkchapman) [inline docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: Once you document the new rights/code-signing stuff you should probably note that use of ,FOLD on a service or function definition within it requires additional rights, and that typically for security reasons ,FOLD should only be enabled in signed modules.

Comment (JamesDeFabia) [inline docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: @richardkchapman I am going to need some more info about the big picture of the rights/signing enhancement. It seems to span several areas. 

Is this PR good for the time being? IOW, can the rest be added at the time signing is added?

Comment (richardkchapman) [inline docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: Yes, probably. I'll make sure there is a JIRA open for documenting the signing and access rights changes.

Comment (richardkchapman) [inline docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: https://track.hpccsystems.com/browse/HPCC-14356

**Pull request merged**